app/views/container_replicator/show.html.haml

```diff
@@ -1,4 +1,6 @@
 - if @display == 'containers'
     = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_nodes'.eql?(@display) && @showtype != "compare"
+    = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
Why the strange `'container_nodes'.eql?(@display)` syntax instead of `@display == 'container_nodes'`?

Also, why the `elsif` instead of adding an or to the first `if`?